android/MobiledgeXSDKDemo/matchingenginehelper/src/main/java/com/mobiledgex/matchingenginehelper/MatchingEngineHelper.java

```diff
@@ -311,23 +311,15 @@ public void startEdgeEvents(boolean newConnection) {
             }
 
             if (mEdgeEventsRunning) {
-                if (me.isAutoMigrateEdgeEventsConnection()) {
-                    if (newConnection) {
-                        message = "Auto-migrating to new app inst";
-                    } else {
-                        message = "Retaining connection to existing app inst";
-                    }
-                } else {
-                    message = "Restarting ServerEdgeEvents";
-                    me.stopEdgeEvents();
-                    me.startEdgeEvents(mEdgeEventsConfig);
-                }
+                message = "Restarting ServerEdgeEvents";
+                me.stopEdgeEvents();
             } else {
                 message = "Subscribed to ServerEdgeEvents";
-                me.startEdgeEvents(mEdgeEventsConfig);
+                Log.i(TAG, "me="+me);

```

Comment:
Do you need to log the whole matching engine or something specific like a edge-events-cookie !

Reply:
Actually just added that log because I was getting an inexplicable NPE. Will remove.